packages/snfoundry/scripts-ts/deploy-contract.ts

```diff
@@ -109,6 +109,24 @@ const deployContract_NotWait = async (payload: {
   }
 };
 
+const findContractFile = (
+  contract: string,
+  fileType: "compiled_contract_class" | "contract_class"
+): string => {
+  const targetDir = path.resolve(__dirname, "../contracts/target/dev");
+  const files = fs.readdirSync(targetDir);
+
+  // Look for files that end with the contract name and file type
+  const pattern = new RegExp(`.*${contract}\\.${fileType}\\.json$`);
+  const matchingFile = files.find((file) => pattern.test(file));
+
+  if (!matchingFile) {
+    throw new Error(`Could not find ${fileType} file for contract ${contract}`);
```

Comment:
might need to leave a suggestion here on what the guy should do if they got this error